marketplace-and-integrations/README.md

```diff
@@ -8,15 +8,28 @@ description: >-
 
 Within this Umbraco DXP (Digital Experience Platform) area of the Umbraco documentation, you'll find topics related to extending Umbraco CMS via packages and integrations.
 
-Umbraco is used in solutions in both a "platform" and "composable" DXP context. With the former, Umbraco has responsibility for both content management and delivery, with extensions added as NuGet packages. In a composable DXP, Umbraco is often part of an overall architecture, focussed on the management of content exposed via APIs for delivery. In both situations, backoffice integrations are an important part of providing an intuitive, unified editor experience.
+Umbraco is used in solutions in both a "platform" and "composable" DXP context. With the former, Umbraco has responsibility for both content management and delivery, with extensions added as NuGet packages. In a composable DXP, Umbraco is often a stand-alone service that takes part in an overall architecture. In these cases, Umbraco CMS is focused on the management of content exposed via APIs for delivery.
 
-Our aim is to enhance the "platform DXP", support the CMS in a "composable DXP", and make it simpler to build integrations with Umbraco.
+Umbraco is also used in a hybrid DXP context. Here, some values are added to the Umbraco platform, like Umbraco Forms or Umbraco Commerce, where Umbraco acts as a component in a wider architecture. In all situations, backoffice integrations are an important part of providing an intuitive, unified editor experience.
```

Comment:
```suggestion
Umbraco is also used in a hybrid DXP context. Here, some services are added directly to the Umbraco platform, like Umbraco Forms or Umbraco Commerce, but Umbraco still acts as a component in a wider architecture.

In all situations, backoffice integrations with third-party services are an important part of providing an intuitive, unified editor experience.
```

Reply:
How about this, to shorten the sentences a bit:

```suggestion
Umbraco is also used in a hybrid DXP context. Here, some services are added directly to the Umbraco platform while Umbraco still acts as a component in a wider architecture. This could be services like Umbraco Forms or Umbraco Commerce.
```

Reply:
Not sure - on reflection we are overusing the word "services" here, so I've changed to "features" in my suggestion above.

If we break out the mention of Forms and Commerce into a separate sentence, it feels like we have to name them as packages again, which we are already referencing in the paragraph above this one.

So overall I think I prefer my modified suggestion above, as it's just there as a passing reference, making it clear what we are talking about when it comes to features in this context.